<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P2PMessenger</class>
 <widget class="QMainWindow" name="P2PMessenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2PMessenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="LoginPage" name="loginPage"/>
      <widget class="ChatPage" name="chatPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginPage</class>
   <extends>QWidget</extends>
   <header>loginpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatPage</class>
   <extends>QWidget</extends>
   <header>chatpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
